lib/runner.ex

```diff
@@ -30,14 +42,16 @@ defmodule Runner do
     modules
     |> Enum.drop_while( &(&1 != start_module))
     |> Enum.take_while( &(run_module(&1) == :passed))
+    |> done?
   end
 
+  defp done?(modules), do: modules == @modules
```

Comment:
That is exactly what I was working on this morning to display a progress bar :)
Build what you need in this branch and I will port my code if need be. 
If I do put together something in the next couple of hours, I'll open a PR for it.